tools/test_cs_equal_db.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python3
+
+
+from util.cassandra_stress import *
+from util.scylla_docker import *
+from cassandra.cluster import Cluster
+
+import argparse
+
+
+### This test populates two tables with cassandra-stress and cql-stress-cassandra-stress.
+### Then it checks whether these two tables are equal.
+def run(args, node: ScyllaDockerNode, cs: CassandraStress):
+    cluster = Cluster([node.ip], port=node.port)
+    session = cluster.connect()
+
+    # Populate two tables in two keyspaces.
+    ks_cassandra = "ks_cassandra"
+    ks_cqlstress = "ks_cqlstress"
+
+    print("\n=== Populating DB... ===\n")
+
+    print(f"\n=== Inserting {args.workload_size} rows to {ks_cassandra}.standard1 with cassandra-stress ===\n")
+    cs.run(command="write",
+           workload_size=args.workload_size,
+           node_ip=node.ip, keyspace=ks_cassandra,
+           threads=args.concurrency)
+
+    print(f"\n=== Inserting {args.workload_size} rows to {ks_cqlstress}.standard1 with cql-stress ===\n")
+    run_cql_stress(command="write",
+                   workload_size=args.workload_size,
+                   node_ip=node.ip, keyspace=ks_cqlstress,
+                   threads=args.concurrency)
+
+    # Check whether two tables are equal.
+    cs_rows = session.execute(f'select * from {ks_cassandra}.standard1')
+    cqlstress_rows = session.execute(f'select * from {ks_cqlstress}.standard1')
+
+    print("\n=== Comparing tables... ===\n")
+    try:
+        if cs_rows != cqlstress_rows:
+            print("Compared tables are not equal!!!")
+            print("\n=== cassandra-stress rows: ===\n")
+            for row in cs_rows:
+                print(row)
+            print("\n=== cql_stress rows: ===\n")
+            for row in cqlstress_rows:
+                print(row)
+            raise AssertionError("Test failed. Tables are not equal.")
+        else:
+            print("\n=== Test successful ===\n")
+    finally:
+        # Drop keyspaces
+        session.execute(f'drop keyspace {ks_cassandra}')
+        session.execute(f'drop keyspace {ks_cqlstress}')
```

Comment:
I think we shouldn't drop those keyspaces. Imagine a developer that runs some test locally, if there is data mismatch then it might be convenient for them to have this data in the database so that they can efficiently query it etc.

To solve the issue of keyspace/table reuse, we can generate keyspace names based on a random number + timestamp and also print it at the beginning of the test.